content/index.md

```diff
@@ -17,7 +17,7 @@ date = "2018-05-22T11:02:24-05:00"
 
 {{% sc_index2 %}}
 #### What is OpenCensus?
-A single distribution of libraries that automatically collects traces and metrics from your app, displays them locally, and sends them to any analysis tool. [{{% sc_gloss1 %}}Watch this video{{% /sc_gloss1 %}}](https://storage.googleapis.com/opencensusio/OpenCensusVideo.mp4) for an overview.  
```

Comment:
Do we want to a link to this video somewhere else?

Reply:
Overview page contains the video as the first thing.